webservice/src/main/java/com/hortonworks/streamline/webservice/StreamlineApplication.java

```diff
@@ -272,6 +272,15 @@ private void registerResources(StreamlineConfiguration configuration, Environmen
             initConfig.put(Constants.CONFIG_AUTHORIZER, authorizer);
             initConfig.put(Constants.CONFIG_SECURITY_CATALOG_SERVICE, securityCatalogService);
             initConfig.put(Constants.CONFIG_SUBJECT, subject);
+            if (configuration.getHttpProxyUrl() == null || configuration.getHttpProxyUrl().isEmpty()) {
+                LOG.warn("No http proxy configuration found. If SAM server is behind an http proxy, please move proxyUrl, proxyUsername and " +
+                        "proxyPassword configuration properties under streams module to httpProxyUrl, httpProxyUsername and httpProxyPassword respectively at" +
+                        " top level in your streamline.yaml");
+            }
```

Comment:
It would be better to either throw an exception and exit or LOG the warning and use these settings if the top level settings are not present.